pymc3/tests/test_shape_handling.py

```diff
@@ -219,3 +220,21 @@ def test_sample_generate_values(fixture_model, fixture_sizes):
         prior = pm.sample_prior_predictive(samples=fixture_sizes)
         for rv in RVs:
             assert prior[rv.name].shape == size + tuple(rv.distribution.shape)
+
+
+def test_observed_with_column_vector():
```

Comment:
Maybe add a comment with the link to the original issue?

Reply:
I'm about to change that test again, because it turned out that #4700 did in fact not fix #4652.
Specifically, `int32`-typed `shapes` still break the RV broadcasting, but this was not covered by the tests.

I will mark these tests as XFAIL, linking to https://github.com/pymc-devs/aesara/issues/390